app/views/anonymous_feedback/problem_reports/index.html.erb

```diff
@@ -0,0 +1,88 @@
+<%= content_for :page_title, "Review Feedback" %>
+<%= content_for :header, "Review Feedback" %>
+
+<%= paginate @feedback, theme: 'twitter-bootstrap-3' %>
+
+<div>
+  <p>Reports rows highlighted in green have already been reviewed.</p>
```

Comment:
Ah yes I could toggle this message to only display when that filter is applied 👍